pkg/controller/storageversionmigrator/storageversionmigrator.go

```diff
@@ -339,6 +355,30 @@ func (svmc *SVMController) sync(ctx context.Context, key string) error {
 	return nil
 }
 
+func isRetriableError(k8sError error) bool {
+	return utilnet.IsConnectionReset(k8sError) ||
+		utilnet.IsHTTP2ConnectionLost(k8sError) ||
+		utilnet.IsProbableEOF(k8sError) ||
+		apierrors.IsServerTimeout(k8sError) ||
+		apierrors.IsTooManyRequests(k8sError) ||
+		apierrors.IsServiceUnavailable(k8sError) ||
+		apierrors.IsInternalError(k8sError) ||
+		apierrors.IsTimeout(k8sError)
+}
+
+func (svmc *SVMController) failMigration(ctx context.Context, toBeProcessedSVM *svmv1alpha1.StorageVersionMigration, err error) error {
```

Comment:
Can we use this in the `// we can't migrate a resource that doesn't exist in the GC` case above?

Reply:
Yep, switched to use the helper.